web/src/main/java/se/inera/intyg/intygsadmin/web/auth/AuthenticationConstansts.java

```diff
@@ -20,13 +20,13 @@
 
 public final class AuthenticationConstansts {
 
-    public static final String LOGIN_URL = "/login";
+    public static final String LOGIN_URL = "/oauth2/authorization/siths";
+    public static final String LOGIN_REDIRECT_URL = "/login/inera";
     public static final String LOGOUT_URL = "/logout";
     public static final String FAKE_PROFILE = "fake";
-    public static final String FAKE_LOGIN_ENDPOINT = "/fake";
+    public static final String FAKE_LOGIN_ENDPOINT = "/fake-api/login";
```

Comment:
Not that it matters much but should we "put the fake login" in testability. /fake-api/login -> /testability/fake (or /testablility/login)

Reply:
We have decided to not change it at this time.